### PR TITLE
CloudAsset: Update asset lib to v1 for ListAssets sample code

### PR DESCRIPTION
**Pull request by peter-zheng-g**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-asset/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

File: samples/snippets/src/main/java/com/example/asset/ListAssetsExample.java [hunk 1]
```diff
@@ -16,15 +16,16 @@
...
 package com.example.asset;
 
+import com.google.cloud.asset.v1.AssetServiceClient;
+import com.google.cloud.asset.v1.AssetServiceClient.ListAssetsPagedResponse;
+import com.google.cloud.asset.v1.ContentType;
+import com.google.cloud.asset.v1.ListAssetsRequest;
+import com.google.cloud.asset.v1.ProjectName;
+import java.util.Arrays;
+
 // [START asset_quickstart_list_assets]
```
Comment (averikitsch) [inline samples/snippets/src/main/java/com/example/asset/ListAssetsExample.java:26]: Import the libraries under this region tag

**Review by averikitsch — APPROVED** [review]

**Comment by peter-zheng-g** [comment]
Not familiar with the process. May I ask what's the blocking issue for this PR? Thanks!

**Comment by averikitsch** [comment]
@Neenu1995 do you know why the integration tests and owlbot are not runnign?

**Comment by Neenu1995** [comment]
Not sure why it is not running on just this PR. Other PRs look fine. Let's try retriggering it.

**Review by Neenu1995 — APPROVED** [review]

**Pull request merged**